src/components/layouts/pages/inspect/Inspect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getPdfFiles } from '../../../../store/pdfFiles/selectors';
+import connect from 'react-redux/es/connect/connect';
+import { Document, Page, pdfjs } from 'react-pdf';
+
+import './Inspect.scss';
+
+pdfjs.GlobalWorkerOptions.workerSrc = '../pdf.worker.js';
+
+function Inspect(props) {
+    const { files } = props;
+
+    const [pageNumber] = useState(1);
+
+    const onLoadSuccess = page => {
+        page.getOperatorList().then(data => {
+            console.log('Bboxes: ', data.argsArray[data.argsArray.length - 1]);
+        });
+    };
+
+    return (
+        <section className="inspect">
+            <Document file={files[0]}>
+                <Page pageNumber={pageNumber} onLoadSuccess={onLoadSuccess} />
+            </Document>
+        </section>
+    );
+}
+
+Inspect.propTypes = {
+    files: PropTypes.array,
```

Comment:
Let's pass one file only into this component, even if in the future we will support multiple we will inspect them one by one. So choose the first file in container instead to keep this one clean.